resolwe_bio/processes/import_data/seq_reads.yml

```diff
@@ -105,6 +105,7 @@
         FASTQC_URL+=("{\"file\":\"fastqc/${NAME}_fastqc/fastqc_report.html\",\"refs\":[\"fastqc/${NAME}_fastqc\"]}")
       done
 
+      re-save-file-list fastq "${FASTQGZ[@]}"
```

Comment:
Is the one in line 66 not enough?

Reply:
The code transcodes the reads quality information if the old encoding is detected. For technical reasons, the output field needs to be referenced again in such cases.